### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mtyton**

### CODE REVIEW

File: ml_service/client/commands/process_data.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        parsed_ents = []
+        for ent in entities:
+            parsed_ents.append([ent.start_char, ent.end_char, ent.label_])
+        parsed_record = deepcopy(record)
+        parsed_record["ents"] = parsed_ents
+        processed_data.append(parsed_record)
+    # send data to database for doccano export
+    count = dt_client.post_doccano_data(processed_data)
+    print(count)
+    # TODO - add logging
```
Comment (mtyton) [inline ml_service/client/commands/process_data.py:24]: will implement this with matching

File: ml_service/nlp/ner.py [hunk 2]
```diff
@@ -54,7 +53,8 @@ def _load_spacy_model(self, use_latest: bool=False):
     def __init__(self, use_latest: bool=False) -> None:
         # first load the proper model
         self.nlp = self._load_spacy_model(use_latest)
-        data = load_and_preprocess(TRAINING_DATA_FILENAME)
+        # TODO - this should be done otherwise
```
Comment (mtyton) [inline ml_service/nlp/ner.py:56]: will be implemnted with matching

File: ml_service/nlp/tools.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+from typing import (
+    List,
+    Tuple,
+    Dict
+)
+import spacy
+import spacy_fastlang
+
+# TODO - add this typing
```
Comment (mtyton) [inline ml_service/nlp/tools.py:9]: remove thins

File: client_api/api/models.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        instance = self._model(**data)
+        db.add(instance)
+        db.commit()
+        db.refresh(instance)
+        return instance
+
+
+@dataclass
+class JobOffer:
+    # TODO - uuid should of type uuid not str
```
Comment (mtyton) [inline client_api/api/models.py:42]: Fix this

**Review by nyckoo — APPROVED** [review]

**Pull request merged**